src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Semantic/SemanticTokensRangeEndpoint.cs

```diff
@@ -24,7 +25,9 @@ internal sealed class SemanticTokensRangeEndpoint(
 
     public void ApplyCapabilities(VSInternalServerCapabilities serverCapabilities, VSInternalClientCapabilities clientCapabilities)
     {
-        _semanticTokensInfoService.ApplyCapabilities(serverCapabilities, clientCapabilities);
+        var legend = new RazorSemanticTokensLegend(clientCapabilities);
+        Capabilities.SemanticTokensOptions(serverCapabilities, legend);
```

Comment:
Content warning: Bikeshedding and/or stream of consciousness comment.

I don't like this line of code. I have no issues with the static method and static class, but I think it works much better in the document color endpoint. I think its because the document color has an expression bodied member, which makes it seem like the method returns something. In this case it just reads weirdly having this method that has no return, but is named without a verb. Would you hate having `Apply` prefixed to the mehod names?

The other option, I suppose, is further clean up of the `RazorSemanticTokensLegend` bits, by having a service that exposes it, and have that service pull in the client capabilities  (via `IInitializeManager`). I guess that's probably the better way to go (and definitely doesn't have to be in this PR).

Reply:
> Would you hate having Apply prefixed to the mehod names?

Not at all. I think that's reasonable